vignettes/Vaccines.Rmd

```diff
@@ -40,153 +123,262 @@ simparams <- get_parameters(list(
-  theme_bw()
+#### Plot clinical incidence and prevalence
 
+```{r, fig.width = 8, fig.height = 4, fig.align = 'center'}
+par(mfrow = c(1,2))
+plot_inci()
+plot_prev()
 ```
+We see a much more gradual decrease in clinical incidence following EPI implementation compared to the mass vaccination strategy.  
+  
+#### Plot doses
+
+You can look at the distribution of doses using the `n_rtss_mass_dose_*` or `n_rtss_epi_dose_*` outputs. It is always a good idea to check the timing of vaccine doses to ensure that it is specified as intended.
+
+```{r, fig.width = 6, fig.height = 4, fig.align = 'center', message=FALSE}
+plot_doses()
+```
+
+### Seasonal mass vaccination
+
+Mass vaccination can also be targeted seasonally. For example, we may want to have a mass vaccination campaign a few months prior to the peak transmission season. In the example below, we will create a parameter set with seasonality and first run a simulation with no vaccine, then run a simulation with a mass vaccination campaign targeting everyone between the ages of 5 months and 15 years.
 
-You can look at the distribution of doses using the n_rtss_mass_dose\_\* outputs:
+We specify that the intervention should use the RTS,S vaccine through the `profile` and `booster_profile` inputs.
 
 ```{r}
-dose_data <- data.frame(timestep = output$timestep,
-                        dose = rep(c("mass 1", "mass 2", "mass 3", "mass booster"), each = length(output$timestep)),
-                        n = c(output$n_rtss_mass_dose_1, output$n_rtss_mass_dose_2, output$n_rtss_mass_dose_3,
-                              output$n_rtss_mass_booster_1))
-dose_data[dose_data$n > 0, ]
+# Use the get_parameters() function to generate a new parameter set with a seasonal profile with malaria incidence in children aged 0-5 rendered in the model output:
+seas_simparams <- get_parameters(
+	list(
+    	human_population = human_population,
+    	incidence_rendering_min_ages = 0,
+    	incidence_rendering_max_ages = 5 * year,
+    	individual_mosquitoes = FALSE,
+    	model_seasonality = TRUE, # Let's try a bi-modal model
+    	g0 = 0.285505,
+    	g = c(-0.325352,-0.0109352,0.0779865),
+    	h = c(-0.132815,0.104675,-0.013919)
+  )
+)
+
+seas_simparams <- set_equilibrium(seas_simparams, starting_EIR)
+
+# Run no vaccine scenario
+output_seas_control <- run_simulation(sim_length *2, seas_simparams)
+```
+
+Next, we will run the scenario with a seasonal mass vaccination campaign implemented starting 3.5 months before the peak of the transmission season.
+```{r}
+# Find the peak seasonality
+peak <- peak_season_offset(seas_simparams)
+
+seasmass_simparams <- set_mass_rtss(
+  parameters = seas_simparams,
+  timesteps = round(year + (peak - month * 3.5), 0),# The vaccination will begin 3.5 months prior to the peak seasonality in the second year.
+  coverages = 1, # 100% of the population between min_ages and max_ages is vaccinated.
+  min_ages = 5 * month, #
+  max_ages = 15 * year,
```

Comment:
again, a little difficult with the high age range here since you are only interested in the outcome of 0-5 year olds

Reply:
When I changed these to 5-17 months, the impact on prevalence was basely visible, so I changed to 5m-5y. Happy to change back to 5-17 months (and maybe remove the prevalence plots then), if you think that's better.